package.json

```diff
@@ -30,52 +30,45 @@
     "eslint-plugin-react": "^7.21.5",
-    "if-env": "^1.0.4",
     "ignore-styles": "^5.0.1",
     "jsdom": "^11.5.1",
-    "lerna": "^2.2.0",
+    "lerna": "^4.0.0",
     "nyc": "^15.0.0",
     "rimraf": "^2.6.1",
     "sinon": "^3.3.0",
     "temp": "^0.8.3",
     "tslint": "^5.12.0",
-    "typedoc": "^0.17.7",
+    "typedoc": "^0.21.2",
     "typedoc-plugin-external-module-map": "^1.2.1",
     "typescript": "^3.9.2",
     "uuid": "^8.0.0"
   },
   "scripts": {
     "preinstall": "node-gyp install",
     "postinstall": "node scripts/post-install.js",
-    "prepare": "yarn prepare:references && yarn --cwd packages/core prepare && yarn prepare:build && yarn prepare:hoisting && yarn download:plugins",
-    "prepare:references": "node scripts/compile-references.js",
-    "prepare:build": "yarn build && run lint && run build \"@theia/example-*\" --stream --parallel",
-    "prepare:hoisting": "theia check:hoisted -s",
-    "clean": "yarn lint:clean && node scripts/run-reverse-topo.js yarn clean",
-    "build": "tsc -b configs/root-compilation.tsconfig.json",
-    "watch": "tsc -b configs/root-compilation.tsconfig.json -w",
-    "lint": "run lint",
+    "prepare": "yarn tsreferences && lerna run prepare && yarn build:ts",
+    "clean": "yarn rebuild:clean && yarn lint:clean && node scripts/run-reverse-topo.js yarn clean",
+    "tsreferences": "node scripts/compile-references.js",
+    "build": "yarn build:ts && yarn build:examples",
+    "build:ts": "tsc -b",
+    "build:examples": "lerna run --scope=\"@theia/example-*\" build --parallel",
+    "watch": "concurrently --kill-others -n tsc,browser,electron -c red,yellow,blue \"tsc -b -w --preserveWatchOutput\" \"yarn --cwd examples/browser watch:bundle\" \"yarn --cwd examples/electron watch:bundle\"",
+    "watch:ts": "tsc -b -w",
+    "lint": "lerna run lint",
     "lint:clean": "rimraf .eslintcache",
     "lint:oneshot": "node --max-old-space-size=4096 node_modules/eslint/bin/eslint.js --cache=true \"{dev-packages,packages,examples}/**/*.{ts,tsx}\"",
-    "docs": "rimraf gh-pages/docs/next && typedoc --tsconfig configs/typedoc-tsconfig.json --options configs/typedoc.json",
-    "test": "yarn test:references && yarn test:theia && yarn test:electron && yarn test:browser",
-    "test:references": "node scripts/compile-references --dry-run",
+    "docs": "rimraf gh-pages/docs/next && typedoc --tsconfig configs/typedoc-tsconfig.json --options configs/typedoc.json .",
+    "test": "yarn test:theia && yarn test:electron && yarn test:browser",
     "test:theia": "run test \"@theia/!(example-)*\" --stream --concurrency=1",
-    "test:browser": "yarn rebuild:browser && run test \"@theia/example-browser\"",
-    "test:electron": "yarn rebuild:electron && run test \"@theia/example-electron\"",
+    "test:browser": "yarn browser test",
+    "test:electron": "yarn electron test",
     "rebuild:clean": "rimraf .browser_modules",
-    "rebuild:browser": "theia rebuild:browser",
```

Comment:
Where did these go?

Reply:
In their respective example applications. I just tweaked the `rebuild` scripts to better handle the use case when it is invoked from non-root locations.

Reply:
So it would be `yarn browser rebuild` now, right.

Reply:
Yes.